andela/ah-titans-frontend
# #160538683 Implements update state from local storage

#### What does this PR do?
Implements update state from local storage

#### Description of tasks to be completed
Checks if a token exists on localStorage and updates the state with the user details found in the token.
If there is no token state is not updated.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ cd ah-titans-frontend
```
```sh
$ git checkout -b ch-store-user-data-in-store-160538683
```
```sh
$ npm install
```
```sh
$ npm start
```
> Navigate to the homepage 

> Open Redux devtools and check the state update action at work

![image](https://user-images.githubusercontent.com/12577659/45688986-74c0a380-bb5b-11e8-880b-3c933b3d273d.png)

> Signin to see the state updated for the currently logged in user

![image](https://user-images.githubusercontent.com/12577659/45689043-991c8000-bb5b-11e8-93ad-1a4d7234212d.png)

![image](https://user-images.githubusercontent.com/12577659/45689138-d84ad100-bb5b-11e8-9dd7-5135f77985b1.png)

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160538683](https://www.pivotaltracker.com/story/show/160538683)
